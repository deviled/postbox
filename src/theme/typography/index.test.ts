import { typography } from '.';

describe('typography', () => {
  it('should match snapshot', () => {
    expect(typography).toMatchSnapshot();
  });
});
