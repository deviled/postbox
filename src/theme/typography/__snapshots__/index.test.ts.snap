// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`typography should match snapshot 1`] = `
Object {
  "fontSize": Object {
    "12": "0.75rem",
    "14": "0.875rem",
    "16": "1rem",
    "18": "1.125rem",
  },
  "fontWeight": Object {
    "bold": 600,
    "light": 200,
    "regular": 400,
  },
}
`;
